docs/_static/css/custom-theme.css

```diff
@@ -18,4 +80,155 @@
 
 .wy-nav-content {
     border-left: 2px #EEEEF1 solid;
-}
\ No newline at end of file
+    max-width: 1200px;
+}
+.caption {
+    border-top: 1px solid #eeeef1;
+}
+.caption-text {
+    font-family: Roboto,Arial,sans-serif;
+    font-weight:500;
+    font-size: 12px;
```

Comment:
I see this class is used in the sidebar as well as for the headings on the front page https://gro-intelligence.github.io/api-client/musenlin-gro-patch-1/index.html

The 12px looks out of place there. Maybe specify 12px for `.wy-menu .caption .caption-text` but 18px for `.toctree-wrapper .caption .caption-text`?

Reply:
I made these changes.
.caption-text {
    font-family: Roboto,Arial,sans-serif;
    font-weight:500;
    font-size: 12px;
    color: #9b9b9e;
    line-height: 14px;
    text-transform: capitalize;
}

.wy-menu-vertical li.toctree-l1.current>a {
    border:none;
    font-size: 16px;
}
<img width="265" alt="Screen Shot 2019-09-16 at 12 22 10 PM" src="https://user-images.githubusercontent.com/55099259/64975280-9f336280-d87c-11e9-88fc-4a82dec7b052.png">

Reply:
I think you did the reverse of what I was saying.

I'm talking about the tiny little headers on the main page here: https://gro-intelligence.github.io/api-client/musenlin-gro-patch-1/index.html

Not the sidebar.